src/app/InteractionModelEngine.h

```diff
@@ -617,6 +632,13 @@ CHIP_ERROR ReadSingleClusterData(const Access::SubjectDescriptor & aSubjectDescr
                                  const ConcreteReadAttributePath & aPath, AttributeReportIBs::Builder & aAttributeReports,
                                  AttributeValueEncoder::AttributeEncodeState * apEncoderState);
 
+/**
+ *  Check whether concreate attribute path is in ember storage.
```

Comment:
```suggestion
 *  Check whether concrete attribute path is an "existent attribute path" in spec terms.
```

It might not be in "ember storage" at all; it might be external. This is just checking for _metadata_, and not even that for the global ones.

Reply:
updated, thanks